src/components/MembersTable.js

```diff
@@ -46,7 +46,19 @@ function MembersTable({ users, refetch}) {
           closeOnDimmerClick={false}
         >
           <Modal.Header>
-            <h2>Member Info</h2>
+                <Grid.Column floated="left">
+                  <h2>Member Info</h2>
+                </Grid.Column>
```

Comment:
Is having member info in a grid column necessary? In the cabinet modal, it isnt used. Both options work. Please choose one for better code consistency.